test/e2e/framework/util.go

```diff
@@ -406,6 +406,16 @@ func SkipUnlessServerVersionGTE(v *utilversion.Version, c discovery.ServerVersio
 	}
 }
 
+func SkipUnlessServerVersionLT(v *utilversion.Version, c discovery.ServerVersionInterface) {
```

Comment:
We have this function cherry-picked into v1.6 codebase when working on an [upgrade test PR](https://github.com/kubernetes/kubernetes/pull/47931).